app/actions/knots.js

```diff
@@ -250,7 +251,7 @@ export function downloadKnot(knot: string) {
       .post(`${baseUrl}/knots/download/`, { knot })
       .then(() => {
         if (process.env.NODE_ENV === 'test') {
-          console.log('Opening url via external browser');
+          // console.log('Opening url via external browser');
```

Comment:
Should we just check if it is not a test env rather than leaving the if block empty?